packages/expo-cli/src/commands/init.ts

```diff
@@ -324,6 +355,54 @@ function logProjectReadyAsync({
   }
 }
 
+async function configureUpdatesProjectFilesAsync(
+  projectRoot: string,
+  initialConfig: BareAppConfig,
+  user: User
+) {
+  const expoPlistPath = path.join(
+    projectRoot,
+    'ios',
+    initialConfig.name,
+    'Supporting',
+    'Expo.plist'
+  );
+  const androidManifestPath = path.join(
+    projectRoot,
+    'android',
+    'app',
+    'src',
+    'main',
+    'AndroidManifest.xml'
+  );
+
+  const appUrl = `https://exp.host/@${user.username}/${initialConfig.name}`;
+  const sdkVersion = (await readConfigJsonAsync(projectRoot)).exp.sdkVersion;
+
+  if (!sdkVersion) {
+    throw new Error('Could not determine project SDK version.');
+  }
+
+  await maybeConfigureUpdatesProjectFileAsync(expoPlistPath, appUrl, sdkVersion);
+  await maybeConfigureUpdatesProjectFileAsync(androidManifestPath, appUrl, sdkVersion);
+}
+
+async function maybeConfigureUpdatesProjectFileAsync(
+  path: string,
+  appUrl: string,
+  sdkVersion: string
+) {
+  if (fs.existsSync(path)) {
```

Comment:
It's best to not check for file existence before reading, because this is not concurrency-safe.

The recommended way is to simply try reading it and then handle the specific error:
```ts
let fileContents;
try {
  fileContents = fs.readFileSync('projects.txt', 'utf8');
} catch (error) {
  if (error.code === 'ENOENT') return; // File doesn't exist
  else throw error;
}
fs.writeFileSync(...);
```